drivers/rpmsg/rpmsg_syslog.c

```diff
@@ -0,0 +1,164 @@
+// SPDX-License-Identifier: GPL-2.0
+/*
+ * Copyright (C) 2017 Pinecone Inc.
+ *
+ * redirect syslog/printf from remote to the kernel.
+ */
+
+#include <linux/module.h>
+#include <linux/rpmsg.h>
+#include <linux/slab.h>
+
+#define NOTIMESTAMP			KERN_TIME
```

Comment:
if this definition useful? I personally find it confusing, I see no relationship between notimestamp and kernel_time - it actually seems like a mutually exclusive concept.

Reply:
Just to fix checkpatch warning:
WARNING: Possible unnecessary KERN_TIME
#110: FILE: drivers/rpmsg/rpmsg_syslog.c:110:
+               pr_info(KERN_TIME "%.*s\n", priv->next, priv->buf);

Could you accept the code with some reasonable chceckpatch warning?